### PR TITLE
Remove globs from entry file imports

### LINKED ISSUE #83: File to import not found or unreadable: active_material/values/*.
I have added in `app/javacsript/packs` folder: `active_admin.js` and `active_admin.scss` file and also added -
`@import "active_material/src/stylesheets/active_material.scss"; `and` import "active_material"` respectively.

While running `bin/webpack-dev-server` command in the terminal for compiling, the following error occurred-
```
ERROR in ./app/javascript/packs/active_admin.scss (./node_modules/css-loader/dist/cjs.js??ref--9-1!./node_modules/postcss-loader/src??ref--9-2!./node_modules/sass-loader/dist/cjs.js??ref--9-3!./app/javascript/packs/active_admin.scss)
Module build failed (from ./node_modules/sass-loader/dist/cjs.js):

@import "active_material/values/*";
^
      File to import not found or unreadable: active_material/values/*.
      in /home/nur-alam/NurAlam/Development/Projects/RubyonRails/welcome_to_australia/node_modules/active_material/src/stylesheets/active_material.scss (line 15, column 1)
Error: 
@import "active_material/values/*";
^
      File to import not found or unreadable: active_material/values/*.
      in /home/nur-alam/NurAlam/Development/Projects/RubyonRails/welcome_to_australia/node_modules/active_material/src/stylesheets/active_material.scss (line 15, column 1)
    at options.error (/home/nur-alam/NurAlam/Development/Projects/RubyonRails/welcome_to_australia/node_modules/node-sass/lib/index.js:291:26)
ℹ ｢wdm｣: Failed to compile.

```

Note: The is a Rails 6 application running with webpacker

### PR DESCRIPTION
**Pull request by kfriend**
Inlines the usage of globs so this project isn't dependent on a custom importer, to enable glob patterns. This fixes #83.

### CODE REVIEW

File: src/stylesheets/active_material.scss [hunk 1]
```diff
@@ -12,25 +12,77 @@
 @import "active_material/values/fonts";
 @import "active_material/values/breakpoints";
 @import "active_material/values/elevation";
-@import "active_material/values/*";
```
Comment (kfriend) [inline src/stylesheets/active_material.scss:15]: Note: All the `value` files were already listed.

File: src/stylesheets/active_material.scss [hunk 2]
```diff
@@ -12,25 +12,77 @@
...
 @import "active_material/values/breakpoints";
 @import "active_material/values/elevation";
-@import "active_material/values/*";
 
 /// Generators describe functions, mixins, and classes
 /// intended for extension. More or less, they are "style builders"
 /// instead of "style descriptors"
 @import "active_material/generators/functions";
 @import "active_material/generators/mixins";
-@import "active_material/generators/*";
```
Comment (kfriend) [inline src/stylesheets/active_material.scss:22]: Note: All the `generator` files were already listed.

**Comment by kfriend** [comment]
@dkniffin Any chance you could take a look at this? Thanks!

**Review by dkniffin — APPROVED** [review]
Looks good!

Sorry it took me so long to review.

**Pull request merged**